code/game/objects/items/teleportation.dm

```diff
@@ -219,3 +219,14 @@
 		if(active_portal_pairs[i] == P)
 			return DESTINATION_PORTAL
 	return FALSE
+
+/obj/item/hand_tele/suicide_act(mob/user)
+	if(iscarbon(user))
+		user.visible_message("<span class='suicide'>[user] is creating a weak portal and sticking their head through! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+		var/mob/living/carbon/itemUser = user
+		var/obj/item/bodypart/head/head = itemUser.get_bodypart(BODY_ZONE_HEAD)
+		head.drop_limb()
+		var/list/safeLevels = SSmapping.levels_by_trait(ZTRAIT_SPACE_RUINS & ZTRAITS_SPACE & ZTRAITS_STATION & ZTRAITS_LAVALAND)

```

Comment:
`SSmapping.levels_by_any_trait(list(ZTRAIT_SPACE_RUINS, ZTRAITS_SPACE, ZTRAITS_STATION, ZTRAITS_LAVALAND))`

Reply:
The space_level trait var isn't a bitflag, which is why this proc is necessary.